proms-resources/frontend/src/main/frontend/src/TOU.json

```
@@ -0,0 +1,80 @@
+    "####Use of your Information by your Health Care Team\\\n",
+    "The information you enter into this Application will be provided to your healthcare team and will become part of your medical record.\\\n",
+    "####What are my rights**\\\n",
+    "You can stop all collection of information by the Application easily by uninstalling the Application. You may use the standard uninstall processes as may be available as part of your mobile device or via the mobile application marketplace or network. You may also close your account through the application.\\\n",
+    "####Data Retention Policy, Managing Your Information\\\n",
+    "We will retain User provided data for an undetermined period of time based on our retention policy. For example. For the purposes of a research trial, your data will be stored for 10 years as per the Records Retention and Destruction Policy at UHN.\\\n",
+    "####Security\\\n",
+    "We are concerned about safeguarding the confidentiality of your information. We provide reasonable physical, electronic, and procedural safeguards to protect information we process and maintain. For example, we reasonably limit access to this information to authorized employees and contractors who need to know that information in order to operate, develop or improve our Application.\\\n",
+    "####Changes\\\n",
+    "This Privacy Policy may be updated from time to time for any reason. We will notify you of any changes to our Privacy Policy by posting the new Privacy Policy here and informing you via the application. You are advised to consult this Privacy Policy regularly for any changes, as continued use is deemed approval of all changes.\\\n",
+    "####Contact us\\\n",
+    "For more detailed information about who sees your information and about your choices contact the Privacy Office.",
+    "UHN Privacy Office:",
+    "Telephone: 416-340-4800 ext. 6937",
+    "Email: privacy@uhn.ca",
+    "Website: http://www.uhn.ca/PatientsFamilies/Patient_Safety_Advocacy/Privacy/Pages/privacy_policy.aspx \\\n",
+    "####Complaints\\\n",
+    "If you have complaints about UHNs compliance with this privacy policy or PHIPA, please contact UHNs Privacy Office at the above contact information. Further questions and concerns can be directed to the Office of the Information and Privacy Commissioner of Ontario at:",
```

Some apostrophes that were in the original text are missing here.

```suggestion
    "If you have complaints about UHN's compliance with this privacy policy or PHIPA, please contact UHN's Privacy Office at the above contact information. Further questions and concerns can be directed to the Office of the Information and Privacy Commissioner of Ontario at:",
```

---

Fixed in 8c43d6e